public/404.html

```diff
@@ -1,56 +1,59 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <title>The page you were looking for doesn't exist (404)</title>
+  <meta http-equiv="refresh" content="5;url=https://mission-list-0401.herokuapp.com/login" />
```

Comment:
想要加上此功能很好，但把 domain 寫死的話，不管是在 local 開發或日後網站部署到其它 domain 時都還是會去 request `https://mission-list-0401.herokuapp.com/login` ，請調整。
```suggestion
  <meta http-equiv="refresh" content="5;url=/login" />
```